frontend/src/App.js

```diff
@@ -12,7 +13,10 @@ function App() {
           <AddEventForm />
         </Route>
         <Route path={'/events'}>
-          <ShowEvents />
+          <EventsPage />
+        </Route>
+        <Route path={'/event/:id'}>
+          <ShowEvent />
         </Route>
```

Comment:
Single line routes are short and sweet ;) 

```suggestion
        <Route path={'/event/:id'} component={ShowEvent}/>
```